### ISSUE TITLE
Permission error on file after TFLINT

### ISSUE THREAD

**Issue by Official-James** [github/super-linter#2433]
## Describe the bug

Once the super-linter (latest) runs a TF scan and we try to run terraform init, we get the following error:
```
Unable to write the module manifest file: open
.terraform/modules/modules.json: permission denied
```
If we use v4.8.5 we do not have the issue. It looks like the latest release is causing this (v4.8.7)

## Expected behavior

When linter runs VALIDATE_TERRAFORM_TFLINT, it should complete and shouldn't cause errors when running terraform init.

## Steps to Reproduce

Workflow steps:

```yml
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Lint Code Base
        uses: github/super-linter@main
        env:
          VALIDATE_TERRAFORM_TFLINT: true

      - uses: hashicorp/setup-terraform@main
        with:
          terraform_version: latest

      - name: Terraform Init
        id: init
        run: terraform init -backend=false
```

## Logs

```
Unable to write the module manifest file: open
.terraform/modules/modules.json: permission denied
```

## Additional context

The error sounds like the file is left open after lint, causing this error (lock file).

**Comment by colwynlegitscript** [comment]
Looks like this is because the Super Linter Docker image uses `root` and now runs `terraform get` which creates the `.terraform` folder. I ran into the same thing when I added a `terraform init` step after the Super Linter step. I ran `ls -la .` just after the Super Linter step and got this.

```
Run ls -la .
total 284
drwxr-xr-x 6 runner docker   4096 Feb  9 22:37 .
drwxr-xr-x 3 runner docker   4096 Feb  9 22:36 ..
drwxr-xr-x 8 runner docker   4096 Feb  9 22:37 .git
drwxr-xr-x 4 runner docker   4096 Feb  9 22:37 .github
-rw-r--r-- 1 runner docker     19 Feb  9 22:37 .gitignore
drwxr-xr-x 3 root   root     4096 Feb  9 22:37 .terraform
-rwxr-xr-x 1 runner docker   1077 Feb  9 22:37 .terraform.lock.hcl
-rw-r--r-- 1 runner docker   9593 Feb  9 22:37 README.md
-rw-r--r-- 1 runner docker     87 Feb  9 22:37 data.tf
-rw-r--r-- 1 runner docker  11844 Feb  9 22:37 main.tf
drwxr-xr-x 4 runner docker   4096 Feb  9 22:37 modules
-rw-r--r-- 1 runner docker    139 Feb  9 22:37 outputs.tf
-rw-r--r-- 1 root   root   217002 Feb  9 22:37 super-linter.log
-rw-r--r-- 1 runner docker   3519 Feb  9 22:37 variables.tf
-rw-r--r-- 1 runner docker    120 Feb  9 22:37 versions.tf
```

I guess a workaround for the time being would be to add a step that runs `rm -rf .terraform` after the Super Linter step. Though in the long run it might make sense to avoid modifying the folder that's being linted. Something super crude that might work is just copying the whole directory to another path, running the linter there, and removing it afterwards. It'd be cool if GitHub Actions had a way to mark the filesystem as read-only for a particular step, though I imagine the need for that feature is pretty small.

**Comment by stuartleeks** [comment]
Should this fix be included in the `slim@v4.9.2` image? (We've just hit the issue with that version)
